persistence/sql/persister_identity.go

```diff
@@ -517,31 +507,36 @@ func (p *Persister) injectTraitsSchemaURL(ctx context.Context, i *identity.Ident
 	return nil
 }
 
-func (p *Persister) getJsonSearchQuery(ctx context.Context, field string, values []string) pop.ScopeFunc {
+func (p *Persister) getJSONSearchQuery(ctx context.Context, field string, values []string) pop.ScopeFunc {
```

Comment:
This code still had unfortunately several SQL injections.

Can you please write an extensive test suite for this? Without it we can not go forward and merge it. It needs to include fuzzing and SQL injection vectors such as `--` and `"` and `'` and other SQL injection entrypoints.

Reply:
Can you add tests with table tests that cover a variety of inputs (potentially using go fuzzing?) to ensure this never ends up in SQL injections?